docs/pages/modules/get-started.mdx

```diff
@@ -44,6 +44,15 @@ If you have an **ios** directory in your project that you created using `expo pr
 > **Note:**
 > If you're using a development client, you need to rebuild your development client any time you want to use new native code.
 
+> **Warning:**
+> If you override the `paths` property in `tsconfig.json` or are using expo version below `48.0.11`, you need to include the following wildcard in [the paths config](https://www.typescriptlang.org/tsconfig#paths).
```

Comment:
```suggestion
> If you override the `paths` property in `tsconfig.json` or are using Expo SDK version below `48.0.11`, you need to include the following wildcard in [the paths config](https://www.typescriptlang.org/tsconfig#paths).
```

One suggestion to either callout Expo SDK or go with package name with inline syntax highlight: `expo`